### ISSUE TITLE
"TypeError: Cannot read property 'length' of undefined" when autocompleting embedded JS on HTML

### ISSUE THREAD

**Issue by spaceporn** [Microsoft/vscode#48173]
- VSCode Version: 1.22.2
- OS Version: Windows 7 x64 SP1

**Steps to Reproduce:**

1. Create an HTML file
2. Use Emmet's `!<tab>` snippet to generate the boilerplate and put a script tag somewhere (I tested inside the body tag)
3. Type `var x = { | }`  inside the script tag, where `|` is where the cursor should be positioned (originally, I got the error using a selector, but trying to debug the error got me here)
4. This is what the document should like after this point:

```
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
</head>

<body>

<script>
    var x = {
        
        }
</script>
</body>

</html>
```

5. Open up quotes, and
6. Error output panel should pop up for the `HTML Language Server` right after opening quotes, this was mine just now:

```
[Error - 10:11:29 PM] Error while computing completions for file:///d%3A/Test/crashtest.html: Cannot read property 'length' of undefined
TypeError: Cannot read property 'length' of undefined
    at getCompletionEntriesFromSymbols (c:\Program Files\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\typescript.js:84803:65)
    at convertStringLiteralCompletions (c:\Program Files\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\typescript.js:84644:21)
    at Object.getCompletionsAtPosition (c:\Program Files\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\typescript.js:84609:23)
    at Object.getCompletionsAtPosition (c:\Program Files\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\typescript.js:103260:35)
    at doComplete (c:\Program Files\Microsoft VS Code\resources\app\extensions\html-language-features\server\out\modes\javascriptMode.js:78:49)
    at Object.<anonymous> (c:\Program Files\Microsoft VS Code\resources\app\extensions\html-language-features\server\out\htmlServerMain.js:320:38)
    at step (c:\Program Files\Microsoft VS Code\resources\app\extensions\html-language-features\server\out\htmlServerMain.js:36:23)
    at Object.next (c:\Program Files\Microsoft VS Code\resources\app\extensions\html-language-features\server\out\htmlServerMain.js:17:53)
    at fulfilled (c:\Program Files\Microsoft VS Code\resources\app\extensions\html-language-features\server\out\htmlServerMain.js:8:58)
    at <anonymous>
```
7. Repeating these steps with `code --disable-extensions -n` returns the same error:

```
[Error - 10:14:11 PM] Error while computing completions for file:///d%3A/Test/noextensions.html: Cannot read property 'length' of undefined
TypeError: Cannot read property 'length' of undefined
    at getCompletionEntriesFromSymbols (c:\Program Files\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\typescript.js:84803:65)
    at convertStringLiteralCompletions (c:\Program Files\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\typescript.js:84644:21)
    at Object.getCompletionsAtPosition (c:\Program Files\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\typescript.js:84609:23)
    at Object.getCompletionsAtPosition (c:\Program Files\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\typescript.js:103260:35)
    at doComplete (c:\Program Files\Microsoft VS Code\resources\app\extensions\html-language-features\server\out\modes\javascriptMode.js:78:49)
    at Object.<anonymous> (c:\Program Files\Microsoft VS Code\resources\app\extensions\html-language-features\server\out\htmlServerMain.js:320:38)
    at step (c:\Program Files\Microsoft VS Code\resources\app\extensions\html-language-features\server\out\htmlServerMain.js:36:23)
    at Object.next (c:\Program Files\Microsoft VS Code\resources\app\extensions\html-language-features\server\out\htmlServerMain.js:17:53)
    at fulfilled (c:\Program Files\Microsoft VS Code\resources\app\extensions\html-language-features\server\out\htmlServerMain.js:8:58)
    at <anonymous>
```

**Does this issue occur when all extensions are disabled?**
Yes, see above.

**Comment by spaceporn** [comment]
Well, I suppose this can be closed since there's mention of a fix on #46939. Sorry for the duplicate, the `is:open` search filter which is on by default might've hidden the related closed issues.

**Comment by mjbvz** [comment]
Yes duplicate of #46939. Should be fixed in insiders
